tests/vcom/base.go

```diff
@@ -0,0 +1,149 @@
+package vcom
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/pkg/namesgenerator"
+	"github.com/dustin/go-humanize"
+	"github.com/lf-edge/eden/pkg/defaults"
+	"github.com/lf-edge/eden/pkg/device"
+	"github.com/lf-edge/eden/pkg/openevec"
+	"github.com/lf-edge/eden/pkg/projects"
+	"github.com/lf-edge/eden/pkg/remote"
+	"github.com/lf-edge/eden/pkg/tests"
+	"github.com/lf-edge/eden/pkg/utils"
+	"golang.org/x/exp/rand"
+)
+
+var (
+	rnode   *remote.RemoteNode
+	appname string
+)
+
+// TestMain is used to provide setup and teardown for the rest of the
+// tests. As part of setup we make sure that context has a slice of
+// EVE instances that we can operate on. For any action, if the instance
+// is not specified explicitly it is assumed to be the first one in the slice
+func initilizeTest(m *testing.M) int {
+	var edgenode *device.Ctx
+	tests.TestArgsParse()
+
+	tc := projects.NewTestContext()
+
+	projectName := fmt.Sprintf("%s_%s", "TestAzureIoT", time.Now())
+
+	// Registering our own project namespace with controller for easy cleanup
+	tc.InitProject(projectName)
+
+	// Create representation of EVE instances (based on the names
```

Comment:
I see that this initialization repeats in every Go tests.
How about to create a framework for eden Go tests, which would handle all this test initialization, minimize boiler-plate and provide various helper methods (what you currently have in pkg/remote)?

Reply:
sure, later I will refactor the test/sec and tests/aziot after this one gets merged.